src/App.js

```diff
@@ -1,27 +0,0 @@
-import logo from './logo.svg';
-import './App.css';
-
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          <code>src/App.js</code>
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
-
-export default App;
```

Comment:
- [ ] Kindly reinstate this component. It helps us keep our code neater if we can import all components into the App component and then import only the App component in the `index.js` file, rather than importing the components directly into the index file which is also used to render the entire app.

- [ ] Kindly import the calculator component here.